manila/share/manager.py

```diff
@@ -2113,6 +2113,23 @@ def create_share_instance(self, context, share_instance_id,
                         share_group=share_group_ref,
                     )
                 )
+            except exception.PortLimitExceeded:
+                with excutils.save_and_reraise_exception():
+                    error = ("Creation of share instance %s failed: "
+                             "failed to allocate network")
+                    LOG.error(error, share_instance_id)
```

Comment:
Just noticed when revisiting: please make this log a warning instead of error. It is something that our clients can solve themselves normally

```suggestion
                    LOG.warning(error, share_instance_id)
```